library/src/main/java/com/airbnb/android/airmapview/DefaultAirMapViewBuilder.java

```diff
@@ -97,12 +95,7 @@ private AirMapViewBuilder getWebMapViewBuilder() {
   }
 
   private static boolean checkNativeMapSupported(Context context) {
-    return isGooglePlayServicesAvailable(context) && hasWriteExternalStoragePermission(context);
-  }
-
-  private static boolean hasWriteExternalStoragePermission(Context context) {
-    return ContextCompat.checkSelfPermission(context,
-        Manifest.permission.WRITE_EXTERNAL_STORAGE) == PackageManager.PERMISSION_GRANTED;
+    return isGooglePlayServicesAvailable(context);
```

Comment:
@felipecsl more info here:
http://developer.android.com/reference/android/Manifest.permission.html#WRITE_EXTERNAL_STORAGE

and I found on reddit about the not needing `WRITE_EXTERNAL_STORAGE` for maps any more. What used to happen is the sdk would check that you had the permission before even trying to load the maps. So it was likely that they just removed that check for 23+ (or 19+) since they can use the `getExternalFilesDir` and `getExternalCacheDir` without needing permissions.